packages/vuelidate/src/index.js

```diff
@@ -29,21 +29,22 @@ export default function useVuelidate (validationsArg, state, registerAs) {
     childResultsKeys.value.push(key)
   }
 
-  const validationResults = setValidations({
-    validations,
+  const validationResults = computed(() => setValidations({
+    validations: unwrap(validations),
     state,
-    childResults
-  })
+    childResults,
+    resultsStorage
+  }))
 
   if (registerAs) {
     injectToParent(validationResults, registerAs)
   }
 
   if (registerAs && childResultsKeys.value.length) {
-    return reactive({
-      ...validationResults,
+    return computed(() => ({
```

Comment:
Wont this mean that instead of unwrapping with `reactive`, we are now returning a computed with `value` instead?

Reply:
I saw you are handling it in the Mixin, but what about normal composition api usage?

Reply:
Oh yeah. In `setup` it then requires us to use `$v.value`. We might want to reiterate on that. The `computed` here was required so that we rerun the validations with new rules if the rules have changed. But I assume we could possibly use a combination of `reactive` and `watch` (where we specifically point at the rules as the watched ref). This could be even more efficient and remove that `.value`.